lib/MooX/Role/RunAlone.pm

```diff
@@ -244,19 +244,19 @@ Examples:
   
  # basic call with retries and progress messages enabled
  my $locked = __PACKAGE__->runalone_lock(
-    attemtps => 3,
+    attempts => 3,
     interval => 2,
     verbose  => 1,
  );
   
  # basic call with retries enabled, but silent
  my $locked = __PACKAGE__->runalone_lock(
-    attemtps => 3,
+    attempts => 3,
     interval => 2,
  );
   
  # make a single (silent) attempt, but return to the caller instead of
- # exiting if the attempt fails. also suppresses any failure message.
+ # exiting if the attempt fails. Also suppresses any failure message.
```

Comment:
This is intentional. I always use all lower-case in code comments unless there is a reason not to such as an abbreviation.